### ISSUE TITLE
Mario + Rabbids: Kingdom Battle crash at startup (Vulkan)

### ISSUE THREAD

**Issue by TheSwagger312** [yuzu-emu/yuzu#6631]
Yuzu Version: EA 1874 and Mainline 686
Cpu: AMD Ryzen 5 2600
GPU: Nvidia Geforce GTX 660 
Ram: 8GB
Pagefile: 25GB
Reproduce: Start the game with Vulkan 
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/6809531/yuzu_log.txt)

**Comment by TheSwagger312** [comment]
Like that?
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/6809968/yuzu_log.txt)

**Comment by v1993** [comment]
Well, the thing is that Nsight dumps are separate files from logs (I really should have clarified that) but I'm not sure where are they stored, so someone who have directly worked with Nsight is needed here.

**Comment by TheSwagger312** [comment]
The nvlddmkm display driver has stopped responding and has been restored.

The description for event ID "13" from source "nvlddmkm" could not be found. Either the component causing this event is not installed on the local computer or the installation is damaged. You can install or repair the component on the local computer.
 
That are two things that I noticed what Windows reports after the crash in the Windows Event display

**Comment by v1993** [comment]
Well, it looks like yuzu somehow crashes part of Nvidia driver (causing loss of device and yuzu's exit in turn), which Windows promptly restarts. The question is what exactly driver doesn't like about data yuzu feeds to it.

**Comment by codbugpatrol** [comment]
@v1993 I have a GTX 670 as well. Regarding your question of which driver, I don't think is the driver, but rather the way keplar cards may work. If any devs have that card, would be nice to simulate the error of device lost and tune the code to work for low end hardwares, maybe is just a simple thing, (and if, lol)

**Comment by BFG1992** [comment]
I have a GTX 750 1GB, and the game had never worked for me on Vulkan, since 2020 I think, when I first discovered Switch emulation. But in older versions it at least booted to logos and the title loading screen, and crashed right before showing the title screen. In the current mainline 1001/EA 2712 it crashes even before that - in the emulator's loading screen, and doesn't reach the game logo screens.

The problem must be with the card itself, I think - I had problems with Vulkan before in many other emulators, including Dolphin, Cemu and RPCS3. For the first two, it was normal gameplay, then 1 second freeze, then rainbow textures on everything, corruptions and other artifacts instead of normal graphics. For the latter, it was crashes in a certain game (Counterspy) when entering a room with lots of enemies in it. For all those issues I found no solutions or even mentions anywhere, so I guess it's either my GPU faulty, or these kind of issues affect only some specific cases (like having a 1 GB Maxwell card or below). I'm not sure how useful this info is, but in case this issue is fixable, I hope it will be useful :)

Also, @ghost , to fix the shadows, disable the "Use asynchronous GPU emulation" checkbox in the Graphics tab of the game properties window.
